src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.cpp

```diff
@@ -39,11 +39,7 @@ FacetSRPDynamicEffector::~FacetSRPDynamicEffector() {
  @return void
  @param callTime  [ns] Time the method is called
 */
-void FacetSRPDynamicEffector::Reset(uint64_t currentSimNanos)
-{
-    if (!this->sunInMsg.isLinked()) {
```

Comment:
Why was this removed?  In Reset we should be checking if required input messages are connect.  Please add this code back.

Reply:
Done